packages/desktop/features/developer-tools.features.ts

```diff
@@ -20,6 +20,9 @@ const developerToolsFeatures: IDeveloperFeatures = {
     deeplink: {
         enabled: true,
     },
+    importErc20Token: {
+        enabled: true,
```

Comment:
is this already ready to be enabled?

Reply:
For now, yes. It is in Developer Tools anyways and we need it to work for a release.